### PR TITLE
Cleaning up test cases suite

### PR DESCRIPTION
**Pull request by WallyMaier**
This PR simply aims to cleanup, fix dates, and remove any "fluff" from the test/regression cases in SU2.  This will not affect the cases in the V&V folder.

Its a purely maintenance pr, while I have some time to be productively unproductive.

- [x] I am submitting my contribution to the develop branch.
- [x] My contribution generates no new compiler warnings (try with --warnlevel=3 when using meson).
- [x] My contribution is commented and consistent with SU2 style (https://su2code.github.io/docs_v7/Style-Guide/).
- [x] I have added a test case that demonstrates my contribution, if necessary.
- [x] I have updated appropriate documentation (Tutorials, Docs Page, config_template.cpp), if necessary.

### CODE REVIEW

File: TestCases/coupled_cht/disc_adj_unsteadyCHT_cylinder/chtMaster.cfg [hunk 1]
```diff
@@ -10,91 +10,61 @@ SOLUTION_FILENAME= solution
...
-%
+
 % ----------------------- DESIGN VARIABLE PARAMETERS --------------------------%
 %
-%DV_KIND= FFD_SETTING
-% First 9 are upper, second 9 are lower DV's
 DV_KIND= FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D,FFD_CONTROL_POINT_2D
-%
-% Marker of the surface in which we are going apply the shape deformation
 DV_MARKER= ( cylinder_fluid, cylinder_solid )
```
Comment (pcarruscag) [inline TestCases/coupled_cht/disc_adj_unsteadyCHT_cylinder/chtMaster.cfg:67]: ^^

File: TestCases/deformation/brick_prism/def_brick_prism.cfg [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
+% File Version 7.4.0 "Blackbird"                                               %
 %                                                                              %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 % ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
 %
-% Physical governing equations (POTENTIAL_FLOW, EULER, NAVIER_STOKES, 
+
 %                               MULTI_SPECIES_NAVIER_STOKES, TWO_PHASE_FLOW, 
 %                               COMBUSTION)
```
Comment (pcarruscag) [inline TestCases/deformation/brick_prism/def_brick_prism.cfg:16]:
```suggestion
```

File: TestCases/deformation/brick_prism_rans/def_brick_prism_rans.cfg [hunk 3]
```diff
@@ -5,13 +5,13 @@
...
+% File Version 7.4.0 "Blackbird"                                               %
 %                                                                              %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 % ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
 %
-% Physical governing equations (POTENTIAL_FLOW, EULER, NAVIER_STOKES, 
+
 %                               MULTI_SPECIES_NAVIER_STOKES, TWO_PHASE_FLOW, 
 %                               COMBUSTION)
```
Comment (pcarruscag) [inline TestCases/deformation/brick_prism_rans/def_brick_prism_rans.cfg:16]:
```suggestion
```

File: TestCases/deformation/brick_pyra/def_brick_pyra.cfg [hunk 4]
```diff
@@ -5,13 +5,13 @@
...
+% File Version 7.4.0 "Blackbird"                                               %
 %                                                                              %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 % ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
 %
-% Physical governing equations (POTENTIAL_FLOW, EULER, NAVIER_STOKES, 
+
 %                               MULTI_SPECIES_NAVIER_STOKES, TWO_PHASE_FLOW, 
 %                               COMBUSTION)
```
Comment (pcarruscag) [inline TestCases/deformation/brick_pyra/def_brick_pyra.cfg:16]: Do a search and delete please :sweat_smile:

**Review by pcarruscag — APPROVED** [review]
Awesome :bouquet:

File: TestCases/aeroelastic/aeroelastic_NACA64A010.cfg [hunk 5]
```diff
@@ -11,290 +11,130 @@
...
-% Time domain
 TIME_DOMAIN=YES
-%
-% Unsteady simulation (NO, TIME_STEPPING, DUAL_TIME_STEPPING-1ST_ORDER, 
-%                      DUAL_TIME_STEPPING-2ND_ORDER, SPECTRAL_METHOD)
 TIME_MARCHING= DUAL_TIME_STEPPING-2ND_ORDER
-%
-% Time Step for dual time stepping simulations (s)
 TIME_STEP= 0.00174532925199
-% 36 steps per period, based on the pitch natural frequency
```
Comment (WallyMaier) [inline TestCases/aeroelastic/aeroelastic_NACA64A010.cfg:70]: Re-add

File: TestCases/aeroelastic/aeroelastic_NACA64A010.cfg [hunk 6]
```diff
@@ -11,290 +11,130 @@
...
-%
-% Natural frequency of the spring in the plunging direction (rad/s)
 PLUNGE_NATURAL_FREQUENCY = 100
-%
-% Natural frequency of the spring in the pitching direction (rad/s)
 PITCH_NATURAL_FREQUENCY = 100
-%
-% The airfoil mass ratio
 AIRFOIL_MASS_RATIO = 60
-%
```
[the remark this answers is not in the document]
Comment (WallyMaier) [reply TestCases/aeroelastic/aeroelastic_NACA64A010.cfg:102]: The sections are applicable for flutter, but they are also standard comments.

Should I re-add standard comments for these "non-standard" problem (aeroelasticity/turbomachinery/etc.)?

File: TestCases/species_transport/venturi_primitive_3species/species3_primitiveVenturi_inletFile.cfg [hunk 7]
```diff
@@ -76,7 +76,7 @@ LINEAR_SOLVER= FGMRES
 LINEAR_SOLVER_PREC= ILU
 LINEAR_SOLVER_ERROR= 1E-8
 LINEAR_SOLVER_ITER= 5
-
+%
```
Comment (WallyMaier) [inline TestCases/species_transport/venturi_primitive_3species/species3_primitiveVenturi_inletFile.cfg:79]: remove

File: TestCases/sliding_interface/single_stage/zone_1.cfg [hunk 8]
```diff
@@ -6,23 +6,13 @@
...
 GRID_MOVEMENT= RIGID_MOTION
-%
-% Motion mach number (non-dimensional). Used for intitializing a viscous flow
-% with the Reynolds number and for computing force coeffs. with dynamic meshes.
 MACH_MOTION= 0.35
-%
-
 MOTION_ORIGIN= 0.0 0.0 0.0
-
-% Angular velocity vector (rad/s) about the motion origi. Example 1250 RPM -> 130.89969389957471 rad/s
```
Comment (WallyMaier) [inline TestCases/sliding_interface/single_stage/zone_1.cfg:26]: re-add

File: TestCases/sliding_interface/pipe/zone_1.cfg [hunk 9]
```diff
@@ -12,16 +12,7 @@
...
 GRID_MOVEMENT= RIGID_MOTION 
-%
-% Motion mach number (non-dimensional). Used for intitializing a viscous flow
-% with the Reynolds number and for computing force coeffs. with dynamic meshes.
 MACH_MOTION= 0.35
-%
-
 MOTION_ORIGIN= 0.0 0.0 0.0
-
-% Angular velocity vector (rad/s) about the motion origi. Example 1250 RPM -> 130.89969389957471 rad/s
```
Comment (WallyMaier) [inline TestCases/sliding_interface/pipe/zone_1.cfg:25]: re-add

File: TestCases/coupled_cht/comp_2d/solid_cylinder2.cfg [hunk 10]
```diff
@@ -7,181 +7,73 @@
...
 
 % ---------------- (SOLIDS) CONDUCTION CONDITION DEFINITION -------------------%
 %
-% We should keep the dimensionalization of the coupled flow solver
 INC_NONDIM= DIMENSIONAL
-%
-% Temperature initialization value
 FREESTREAM_TEMPERATURE= 350.0
-%
-% Nettis case: hollow cylinder (air w/ 4x the conductivity)
```
Comment (WallyMaier) [inline TestCases/coupled_cht/comp_2d/solid_cylinder2.cfg:53]: re-add

File: TestCases/coupled_cht/comp_2d/solid_cylinder3.cfg [hunk 11]
```diff
@@ -7,181 +7,72 @@
...
 
 % ---------------- (SOLIDS) CONDUCTION CONDITION DEFINITION -------------------%
 %
-% We should keep the dimensionalization of the coupled flow solver
 INC_NONDIM= DIMENSIONAL
-%
-% Temperature initialization value
 FREESTREAM_TEMPERATURE= 350.0
-%
-% Nettis case: hollow cylinder (air w/ 4x the conductivity)
```
Comment (WallyMaier) [inline TestCases/coupled_cht/comp_2d/solid_cylinder3.cfg:53]: re-add

File: TestCases/coupled_cht/incomp_2d/cht_2d_3cylinders.cfg [hunk 12]
```diff
@@ -9,74 +9,22 @@
...
-% Number of total iterations
 OUTER_ITER = 11
-%
-% Mesh input file
 MESH_FILENAME= mesh_cht_3cyl.su2
-%
-% Mesh input file format (SU2, CGNS, NETCDF_ASCII)
 MESH_FORMAT= SU2
 
-% These are just default parameters so that we can run SU2_DOT_AD, they have no physical meaning for this test case.
```
Comment (WallyMaier) [inline TestCases/coupled_cht/incomp_2d/cht_2d_3cylinders.cfg:45]: re-add

File: TestCases/coupled_cht/incomp_2d_unsteady/solid_cylinder3.cfg [hunk 13]
```diff
@@ -7,82 +7,38 @@
...
-%
-% Marker(s) of the surface to be plotted or designed
 MARKER_PLOTTING= ( cylinder_inner3 )
-%
-% Marker(s) of the surface where the functional (Cd, Cl, etc.) will be evaluated
 MARKER_MONITORING= ( NONE )
 
 % ---------------- (SOLIDS) CONDUCTION CONDITION DEFINITION -------------------%
 %
-% We should keep the dimensionalization of the coupled flow solver
```
Comment (WallyMaier) [inline TestCases/coupled_cht/incomp_2d_unsteady/solid_cylinder3.cfg:47]: re-add

File: TestCases/unsteady/plunging_naca0012/plunging_NACA0012.cfg [hunk 14]
```diff
@@ -11,236 +11,106 @@
...
 % ------------------------- UNSTEADY SIMULATION -------------------------------%
 %
-% Unsteady simulation (NO, TIME_STEPPING, DUAL_TIME_STEPPING-1ST_ORDER, 
-%                      DUAL_TIME_STEPPING-2ND_ORDER)
 TIME_MARCHING= DUAL_TIME_STEPPING-2ND_ORDER
-%
-% Time Step for dual time stepping simulations (s)
 TIME_STEP= 0.0023555025613149587
-% 24 steps per period: 0.0024536485013697488
-% 25 steps per period: 0.0023555025613149587
```
Comment (WallyMaier) [inline TestCases/unsteady/plunging_naca0012/plunging_NACA0012.cfg:37]: re-add

File: TestCases/unsteady/pitching_naca64a010_rans/turb_NACA64A010.cfg [hunk 15]
```diff
@@ -5,392 +5,153 @@
...
 % ------------------------- UNSTEADY SIMULATION -------------------------------%
 %
-% Unsteady simulation (NO, TIME_STEPPING, DUAL_TIME_STEPPING-1ST_ORDER, 
-%                      DUAL_TIME_STEPPING-2ND_ORDER)
 TIME_MARCHING= DUAL_TIME_STEPPING-2ND_ORDER
-%
-% Time Step for dual time stepping simulations (s)
 TIME_STEP= 0.0023555025613149587
-% 24 steps per period: 0.0024536485013697488
-% 25 steps per period: 0.0023555025613149587
```
Comment (WallyMaier) [inline TestCases/unsteady/pitching_naca64a010_rans/turb_NACA64A010.cfg:37]: re-add

File: TestCases/unsteady/pitching_naca64a010_euler/pitching_NACA64A010.cfg [hunk 16]
```diff
@@ -11,360 +11,140 @@
...
 
 % ------------------------- UNSTEADY SIMULATION -------------------------------%
 %
-% Unsteady simulation (NO, TIME_STEPPING, DUAL_TIME_STEPPING-1ST_ORDER, 
-%                      DUAL_TIME_STEPPING-2ND_ORDER)
 TIME_MARCHING= DUAL_TIME_STEPPING-2ND_ORDER
-%
-% Time Step for dual time stepping simulations (s)
 TIME_STEP= 0.0023555025613149587
-% 24 steps per period: 0.0024536485013697488
```
Comment (WallyMaier) [inline TestCases/unsteady/pitching_naca64a010_euler/pitching_NACA64A010.cfg:36]: re-add

File: TestCases/unsteady/pitching_naca64a010_euler/pitching_NACA64A010.cfg [hunk 17]
```diff
@@ -11,360 +11,142 @@
...
 TIME_MARCHING= DUAL_TIME_STEPPING-2ND_ORDER
-%
-% Time Step for dual time stepping simulations (s)
 TIME_STEP= 0.0023555025613149587
 % 24 steps per period: 0.0024536485013697488
-% 25 steps per period: 0.0023555025613149587
 %
-% Total Physical Time for dual time stepping simulations (s)
 MAX_TIME= 0.59
-% 10 periods: 0.5888756403287397
```
Comment (pcarruscag) [inline TestCases/unsteady/pitching_naca64a010_euler/pitching_NACA64A010.cfg:41]: Yes for these unusual cases it may make sense to keep more comments.

Comment (WallyMaier) [reply TestCases/unsteady/pitching_naca64a010_euler/pitching_NACA64A010.cfg:41]: 👍

**Pull request merged**